charts/osdfir-infrastructure/charts/openrelik/values.yaml

```diff
@@ -0,0 +1,312 @@
+    ##
+    repository: ghcr.io/openrelik/openrelik-server
+    ## @param api.image.pullPolicy OpenRelik image pull policy
+    ## ref https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy
+    ##
+    pullPolicy: IfNotPresent
+    ## @param api.image.tag Overrides the image tag whose default is the chart appVersion
+    ##
+    tag: "2024.12.12"
+  ## OpenRelik api resource requests and limits
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param api.resources.limits Resource limits for the API container
+  ## @param api.resources.requests Requested resources for the API container
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 500m
+    ##    memory: 1Gi
+    limits: {}
+    ## Example:
+    ## requests:
+    ##    cpu: 500m
+    ##    memory: 1Gi
+    requests: {}
+## @section OpenRelik Mediator configuration
+##
+mediator:
+  image:
+    ## @param mediator.image.repository OpenRelik Mediator image repository
+    ##
+    repository: ghcr.io/openrelik/openrelik-mediator
+    ## @param mediator.image.pullPolicy OpenRelik image pull policy
+    ## ref https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy
+    ##
+    pullPolicy: IfNotPresent
+    ## @param mediator.image.tag Overrides the image tag whose default is the chart appVersion
+    ##
+    tag: "2024.12.12"
+  ## OpenRelik mediator resource requests and limits
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param mediator.resources.limits Resource limits for the Mediator container
+  ## @param mediator.resources.requests Requested resources for the Mediator container
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 500m
+    ##    memory: 1Gi
+    limits: {}
+    ## Example:
+    ## requests:
+    ##    cpu: 500m
+    ##    memory: 1Gi
+    requests: {}
+## @section OpenRelik Metrics configuration
+##
+metrics:
+  image:
+    ## @param metrics.image.repository OpenRelik Metrics image repository
+    ##
+    repository: ghcr.io/openrelik/openrelik-metrics
+    ## @param metrics.image.pullPolicy OpenRelik image pull policy
+    ## ref https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy
+    ##
+    pullPolicy: IfNotPresent
+    ## @param metrics.image.tag Overrides the image tag whose default is the chart appVersion
+    ##
+    tag: "2024.12.12"
+  ## OpenRelik metrics resource requests and limits
+  ## ref: https://kubernetes.io/docs/user-guide/compute-resources/
+  ## @param metrics.resources.limits Resource limits for the Metrics container
+  ## @param metrics.resources.requests Requested resources for the Metrics container
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 500m
+    ##    memory: 1Gi
+    limits: {}
+    ## Example:
+    ## requests:
+    ##    cpu: 500m
+    ##    memory: 1Gi
+    requests: {}
+## @section OpenRelik Worker Configurations
+##
+workers:
```

Comment:
add env key/values cofiguration here? for each worker so they can get configured in the worker template? 

Instead of hardcoding them into the template (e.g. for Timesketch) and wrapping them in if statements.

Reply:
we probably also want to be able to set resources per worker in the values.yaml. As a plaso worker will be configured with different resources form a strings worker.

Reply:
Done. I kept the timesketch code as is but allowed env variables to be able to specify for the rest.

So the workers do take resources, but I see its not fully clear on how so I added both `env` and `resources` as empty variables to be configured and added documentation above on how to use them. I also added a note on how the TImesketch integration works above the timesketch worker